netconf/transport_junos.go

```diff
@@ -18,6 +18,7 @@ type TransportJunos struct {
 // Close closes an existing local NETCONF session.
 func (t *TransportJunos) Close() error {
 	if t.cmd != nil {
+		t.cmd.Wait()
```

Comment:
Shoudln't the transport be closed first (usually triggering the application to be closed)

Reply:
Sorry, I didn't see this comment previously.

When running without this new line we were seeing lots of defunct processes on the routers, however with this we no longer see it. We have been running with this patched in production since this PR was submitted.

Can you elaborate on your comment; are you saying the code is correct as it is, or suggesting something else rather than  `t.cmd.Wait()`?